.github/workflows/pre-release.yml

```
@@ -0,0 +1,50 @@
+name: prerelease
+on:
+  push:
+    branches:
+      # releases/<tag>/<version>
+      - releases/*/*
+jobs:
+  prerelease:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/setup-node@v2
+        with:
+          node-version: '*'
+          cache: 'npm'
+          check-latest: true
+          cache-dependency-path: 'npm-shrinkwrap.json'
+          registry-url: 'https://registry.npmjs.org'
+      # This can be removed once npm >=7.20.0 is installed by default.
+      # See https://github.com/npm/cli/issues/2755.
+      - name: Upgrade npm
+        run: npm install -g npm@latest
+      - name: Install core dependencies
+        run: npm ci --no-audit
```

```suggestion
    steps:
      - uses: actions/checkout@v2
      - uses: actions/setup-node@v2
        with:
          node-version: '*'
          cache: 'npm'
          check-latest: true
          registry-url: 'https://registry.npmjs.org'
```

We should match the other steps in this repo workflows

---

Oops! Is this ok now after eedae1c09555c17d6ad883c87d1641afb91e1af8?